pulsar-broker-common/src/main/java/org/apache/pulsar/bookie/rackawareness/BookieRackAffinityMapping.java

```diff
@@ -80,7 +80,7 @@ public static MetadataStore createMetadataStore(Configuration conf) throws Metad
             store = (MetadataStore) storeProperty;
         } else {
             String url;
-            String metadataServiceUri = ConfigurationStringUtil.castToString(conf.getProperty("metadataServiceUri"));
+            String metadataServiceUri = ConfigurationStringUtil.castToString(conf.getProperty("metadataStoreUrl"));
```

Comment:
It will introduce a compatibility problem, the user upgrades this pr, it can't read the rack info.

Reply:
Fixed, first get from the broker side，then downgrade to the old logic(bk side ).